src/gql/queries/last-mainline-commit.graphql

```diff
@@ -22,6 +22,7 @@ query LastMainlineCommit(
           }
         }
         id
+        order
```

Comment:
You can get the `order` from a task as well (in case that makes `passingOrderNumber` calculation easier i.e. `lastPassingTask.order`). Just something I wanted to call out!